javascript/javascript1/week3/smart-ease.js

```diff
@@ -0,0 +1,153 @@
+//Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+
+const nameToRemove = "Ahmad";
+names.splice(1,1);  
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+
+//When will we be there??
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+function getThereFaster() {
+    //calculation of the required speed to arrive the destination in 3 hours and 34 minutes
+    const speficySpeed =(travelInformation.destinationDistance/(3+(34/60)));
+    travelInformation.speed =speficySpeed; 
```

Comment:
You should take the speed from travelInformation object as you did for destinationDistance, no need to calculate the speed. Try to read the assignment again.
`Write a function where you specify your speed in km/h and how far you have to go in km. The function has to return the time it will take to arrive at your destination.`